backend/capellacollab/sessions/files/routes.py

```diff
@@ -90,3 +91,23 @@ def upload_files(
     OPERATOR.upload_files(id, tar_bytes)
 
     return {"message": "Upload successful"}
+
+
+@router.get(
+    "/download",
+)
```

Comment:
A response_model is missing here.

Reply:
Does that work, for a streaming response object?